demo/src/main/java/com/example/demo/CreateTeams.java

```diff
@@ -0,0 +1,27 @@
+package com.example.demo;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CreateTeams {
```

Comment:
Consider making this a spring component + remove static accessors and use dependency injection instead